### DISCUSSION TITLE
Community Docker GitHub Team

### DISCUSSION THREAD

**Discussion by PeerRich** [calcom/docker#32] [Announcements]
Hey everyone, we've been super super busy with our core product which is why the /docker repository didn't receive the attention it deserved.

We're happy to announce that we've now launched our very own GitHub Docker team, which you can join to help maintain the repository: https://github.com/orgs/calcom/teams/docker

We're also looking to compensate active members of the community 🙏 

Please comment if you want to be added!

**Comment by krumware** [comment]
👋 👍

**Reply by PeerRich** [reply]
added

**Reply by krumware** [reply]
thanks!

**Comment by kelvinalfaro** [comment]
I'd be interested.

**Reply by PeerRich** [reply]
added

**Comment by AmoghSaxena** [comment]
I am interested. :+1:

**Reply by PeerRich** [reply]
added

**Comment by MR-Os1** [comment]
👍👍👍 interested

**Reply by PeerRich** [reply]
added

**Comment by justmiles** [comment]
:+1: Happy to help here

**Reply by PeerRich** [reply]
added!

**Comment by arnabtarwani** [comment]
I'm interested!

**Reply by PeerRich** [reply]
added!

**Comment by btsuhako** [comment]
interested! and can also help with deploying to Kubernetes

**Reply by PeerRich** [reply]
added!

**Comment by nmogbuk** [comment]
I am interested.

**Reply by PeerRich** [reply]
added

**Comment by mangkoran** [comment]
I'd like to help too!

**Reply by PeerRich** [reply]
added

**Comment by Zodyart** [comment]
The install instructions are outdated, can not install calcom by following instructions please revisit and fix. Will appreciate very much!!! Thanx!!!!

**Comment by krumware** [comment]
@Zodyart are you talking about the cal.com instructions or the docker instructions? Please open an issue with more detail if you're having trouble

**Comment by avinal** [comment]
I would like to help too 😄

**Reply by PeerRich** [reply]
added

**Comment by smallw00d2211** [comment]
I would love to help

**Reply by PeerRich** [reply]
added

**Comment by erzhan89** [comment]
I would like to help too.

**Reply by PeerRich** [reply]
added

**Comment by Leopere** [comment]
Can someone update the original post to actually link to a non 404 url?

**Reply by PeerRich** [reply]
fixed

**Reply by PeerRich** [reply]
also added to the team. sorry for the wait

**Comment by PeerRich** [comment]
thank you all! everyone here is doing gods work – because I have no idea what's happening with regards to docker

<img width="503" alt="CleanShot 2022-08-09 at 15 47 40@2x" src="https://user-images.githubusercontent.com/8019099/183665823-c4b18378-762f-470c-9c1a-403a9b04ef76.png">

**Reply by avinal** [reply]
Hehe, don't worry we will figure it out together 🤝

**Comment by EverettBerry** [comment]
Happy to help!

**Reply by PeerRich** [reply]
added

**Comment by amlcodes** [comment]
happy to help!

**Reply by PeerRich** [reply]
added

**Comment by ramessesii2** [comment]
I'd love to help 😸

**Reply by PeerRich** [reply]
added

**Comment by GQAdonis** [comment]
I would like to help.  Could provide Terraform scripts.

**Reply by PeerRich** [reply]
added

**Comment by xoda** [comment]
Happy to help, if you're still looking for someone. 

Please note that https://github.com/orgs/calcom/teams/docker is returning 404, I guess you'll need to update it

**Reply by PeerRich** [reply]
added. i think its a 404 until you are in the team

**Reply by xoda** [reply]
> added. i think its a 404 until you are in the team

@PeerRich Still 404!

**Reply by PeerRich** [reply]
thats weird. working for me. will look into this eventually (not a priority right now)

**Reply by xoda** [reply]
<img src=https://user-images.githubusercontent.com/5850473/190465637-a71ba839-8088-47c3-b56a-f21510931649.png width=400>

@PeerRich that's why :)

Do you mind to try again?

**Comment by mavishay** [comment]
I want to help,
If help still needed.

**Reply by Leopere** [reply]
Reach out to me on slack under the same username as here.

**Comment by Haliax** [comment]


**Reply by Leopere** [reply]
Reach out to me on slack under the same username as here.

**Comment by Leopere** [comment]
> Hey everyone, we've been super super busy with our core product which is why the /docker repository didn't receive the attention it deserved.
> 
> We're happy to announce that we've now launched our very own GitHub Docker team, which you can join to help maintain the repository: https://github.com/orgs/calcom/teams/docker
> 
> We're also looking to compensate active members of the community 🙏
> 
> Please comment if you want to be added!

I really just needed a tester and my huge rewrite wouldn't have been trash.

**Reply by AnsenIO** [reply]
hi can help with testing

**Reply by Leopere** [reply]
I mostly need someone who knows a bit more about the app on a normal operational basis than me.  Then I can containerize what they do to launch it.  I got a mostly drop in replacement ready container finished but then couldn't recruit any testers from Cal.com.  At the time it was a bit tricky to distract them with this project from what they were doing for the recent release.

**Comment by criadoperez** [comment]
Also willing to participate.

**Reply by Leopere** [reply]
Reach out to me on slack under the same username as here.

**Comment by AnsenIO** [comment]
> Hey everyone, we've been super super busy with our core product which is why the /docker repository didn't receive the attention it deserved.
> 
> We're happy to announce that we've now launched our very own GitHub Docker team, which you can join to help maintain the repository: https://github.com/orgs/calcom/teams/docker
> 
> We're also looking to compensate active members of the community pray
> 
> Please comment if you want to be added!

hi, i would like to help with the docker image (and finally on Kubernetes since my infrastructure runs on it)

the link above is not working for me, this other link works for me (for teams only?): https://github.com/calcom/docker  is it the same repo?

**Comment by AuthorShin** [comment]
@PeerRich 
I can also help with some part of the docker project part, mostly testing like in alpha level.
